test/run.sh

```diff
@@ -37,6 +39,8 @@ function main() {
           --acceptance-go-client-named-vectors|-agnv) run_all_tests=false; run_acceptance_go_client=false; run_acceptance_go_client_named_vectors=true;;
           --acceptance-only-graphql|-aog) run_all_tests=false; run_acceptance_graphql_tests=true ;;
           --acceptance-only-replication|-aor) run_all_tests=false; run_acceptance_replication_tests=true ;;
+          --acceptance-only-maintenance|-aom) run_all_tests=false; run_acceptance_maintenance_tests=true ;;
```

Comment:
thanks for adding `--only-acceptance-*` option so I think that this dedicated option only for maintenance package can removed